developer_guides/unit_tests.rst

```diff
@@ -56,7 +56,7 @@ Preparing cmocka package
 
 1. Build cmocka with static library on:
 
-   .. code-block:: bash
+.. code-block:: bash

```

Comment:
@deb-intel outdenting all these code blocks will cause the numbered list to reset to 1 for the next item.  The indented code block directive that was there was correct since the code block is part of the list item material.